src/commands/registerCommands.ts

```diff
@@ -138,7 +140,7 @@ export function registerCommands(): void {
     registerCommand('vscode-docker.context.use', useDockerContext);
     registerCommand('vscode-docker.context.inspect', inspectDockerContext);
     registerCommand('vscode-docker.context.remove', removeDockerContext);
-    registerCommand('vscode-docker.installDocker', installDocker);
+    registerWorkspaceCommand('vscode-docker.installDocker', installDocker);
```

Comment:
Unrelated, I think this needs to be set up as a workspace command

Reply:
registerWorkspaceCommand ensured the command working on the local host?

Reply:
If I understand it correctly, yes. I want to take some time to investigate and be sure.